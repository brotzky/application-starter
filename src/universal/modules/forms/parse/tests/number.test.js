import number from '../number';

describe('[Form Field Parsing] number', () => {
  describe('number()', () => {
    it('should return an integer from a string', () => {
      expect(number('123')).toMatchSnapshot();
    });
  });
});
