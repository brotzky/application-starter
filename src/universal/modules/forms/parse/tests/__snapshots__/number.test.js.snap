// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Form Field Parsing] number number() should return an integer from a string 1`] = `123`;
